packages/calcite-components/src/utils/sortableComponent.ts

```diff
@@ -60,19 +60,29 @@ export interface SortableComponent {
   canPut: (detail: DragDetail) => boolean;
 
   /**
-   * Called by any change to the list (add / update / remove).
+   * Called when any sortable component drag starts.
    */
-  onDragSort: (detail: DragDetail) => void;
+  onGlobalDragStart: () => void;
```

Comment:
Are the global drag start/end callbacks for internal purposes? If so, maybe we could doc that any public drag events should emit within `onDragStart`/`onDragEnd`?